nifi-extension-bundles/nifi-hashicorp-vault-bundle/nifi-hashicorp-vault-client-service-api/src/main/java/org/apache/nifi/vault/hashicorp/HashiCorpVaultClientService.java

```diff
@@ -37,17 +38,15 @@ public interface HashiCorpVaultClientService extends ControllerService, Verifiab
             "Use one or more '.properties' files to configure the client");
 
     PropertyDescriptor CONFIGURATION_STRATEGY = new PropertyDescriptor.Builder()
-            .displayName("Configuration Strategy")
-            .name("configuration-strategy")
+            .name("Configuration Strategy")
             .required(true)
             .allowableValues(DIRECT_PROPERTIES, PROPERTIES_FILES)
             .defaultValue(DIRECT_PROPERTIES.getValue())
             .description("Specifies the source of the configuration properties.")
             .build();
 
     PropertyDescriptor VAULT_URI = new PropertyDescriptor.Builder()
-            .name("vault.uri")
-            .displayName("Vault URI")
+            .name("Vault URI")
```

Comment:
In this particular case, the property names align to the underlying library, so this particular class should remain unchanged.